docs/src/python/user-guide/misc/visualization.py

```diff
@@ -3,30 +3,33 @@
 
 path = "docs/data/iris.csv"
 
-df = pl.scan_csv(path).group_by("species").agg(pl.col("petal_length").mean()).collect()
+df = pl.read_csv(path)
```

Comment:
I've replaced the current example with a more colourful one

A couple of screenshots:

![Screenshot 2024-08-11 160500](https://github.com/user-attachments/assets/89d26c6a-5cd9-469c-b3ab-37e02d05271d)

![Screenshot 2024-08-11 160525](https://github.com/user-attachments/assets/0216f706-ad04-42cd-bca6-aa0f0f153a53)